### ISSUE TITLE
Error when try to run quick start demo

### ISSUE THREAD

**Issue by littlejin** [dmlc/difacto#11]
Hello, when I try to run quick start demo, I came to this

```
[root@algo064 difacto]# build/difacto data_in=data/gisette_scale val_data=data/gisette_scale.t lr=.02 V_dim=2 V_lr=.001 batch_size=1000
[11:28:34] src/main.cc:27: Unrecognized keyword argument for task = train
[11:28:34] src/main.cc:29:  - val_data = data/gisette_scale.t
[11:28:34] src/sgd/sgd_learner.cc:35: Start epoch 0
[11:28:34] ./src/common/spmm.h:177: Check failed: (sp + k) <= (max_len) 
Aborted
```
Any idea how to fix that ?

**Comment by mli** [comment]
should be fixed by #10

**Comment by littlejin** [comment]
It works, issue close
